L_ZWay2.lua

```diff
@@ -781,12 +790,18 @@ local command_class = {
   ["50"] = function (d, inst, meta)
     local var = (inst.metrics.scaleTitle or '?'): upper ()
     local translate = {W = "Watts", A = "Amps", V = "Volts"}      -- 2020.02.10 thanks @rafale77
-    if var then setVar (translate[var] or var, inst.metrics.level, meta.service, d) end
+    if var then
+    setVar (translate[var] or var, inst.metrics.level, meta.service, d)
+      if var == "KWH" then
+        setVar ("KWHReading", inst.updateTime, meta.service, d)
+      end
+    end
   end,
 
   -- door lock
   ["98"] = function (d, inst)
-    setVar ("Status",open_or_close (inst.metrics.level), SID.DoorLock, d)
+      setVar ("Status",open_or_close (inst.metrics.level), SID.DoorLock, d)
+      setVar ("LastTrip", inst.updateTime, SID.SecuritySensor, d)
```

Comment:
I can't believe the choice of Vera not to use the `SecuritySensor` service for `Status`.

Is the LastTrip actually put into that service?  If so, then the `Status` should also be mirrored there?  I don't have a door lock, so cant find out except to look at the device/service files.  I do see that there is a `dateTime` variable defined in the `DoorLock` service - should this, in fact, be what is set here?  

I also note that the device file does not specify use of the `SecuritySensor` service!

Either way, for reasons discussed elsehwere, `inst.updateTime` should not be used, but `os.time()` instead.